src/controllers/email.ts

```diff
@@ -0,0 +1,32 @@
+import nodemailer from 'nodemailer';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+/**
+ * Sends an email using nodemailer.
+ * @returns {Promise<void>} A promise that resolves when the email is sent.
+ */
+export async function sendEmail() {
+  const transporter = nodemailer.createTransport({
+    host: 'smtp.gmail.com',
+    port: 465,
+    secure: true,
+    auth: {
+      user: process.env.USER,
+      pass: process.env.PASS,
+    },
+  });
+
+  const info = await transporter.sendMail({
+    from: '"ALPHA Atlp Cohort 29" <musabemusabe10@gmail.com>',
+    to: 'musabehonore@gmail.com',
+    subject: 'Your email verification',
+    html: `
+    <h1>Y'ello there You just signed up somewhere ....</h1>
+
```

Comment:
wrong implementation create File like in utils that handles everything and create html/css file that have good format of email that represent every.

Reply:
Yeah